libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -158,7 +157,6 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
             block_hash,
           },
         }),
-        transformResponse: JSONBigNumber.parse,
```

Comment:
according to JIRA task "backend have returned string in the API".
it doesn't seem true at all when I look at production endpoints

if you look for instance at a 
`https://explorers.api.live.ledger.com/blockchain/v3/eth/addresses/<ADDRESS>/transactions`

I can see that `transfer_events.list` still have events with `count` being a number

or more simply, the API

`https://explorers.api.live.ledger.com/blockchain/v3/eth/addresses/<ADDRESS>/balance`

still returns me a number.

Reply:
Quick answer to this is that changes are only on staging not even on prod yet. But you're right on other points.